src/generator/templates/sample.njk

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 Google LLC
```

Comment:
Don't we have a license in a separate included `.njk` as we do in gapic-generator-typescript?
```njk
{% import "_license.njk" as license -%}
{% license.license() -%}
```

Reply:
For you #2057